### PR TITLE
fix: accept 2xx status instead of only 200

### PR DESCRIPTION
**Pull request by ytvnr**
**Issue**

https://gravitee.atlassian.net/browse/APIM-1992

---
**Gravitee.io Automatic Deployment**

🚀 A prerelease version of this package has been published on Gravitee's private artifactory, you can:
 - use it directly by updating your project with version: `1.1.3-apim-1992-webhook-success-2xx-SNAPSHOT`
 - download it from Artifactory [here](https://odbxikk7vo-artifactory.services.clever-cloud.com/gravitee-snapshots/io/gravitee/notifier/gravitee-notifier-webhook/1.1.3-apim-1992-webhook-success-2xx-SNAPSHOT/gravitee-notifier-webhook-1.1.3-apim-1992-webhook-success-2xx-SNAPSHOT.zip)

### CODE REVIEW

File: src/test/java/io/gravitee/notifier/webhook/AbstractWebhookNotifier.java [hunk 1]
```diff
@@ -36,6 +36,18 @@ protected static BiFunction<Void, Throwable, Object> success(VertxTestContext te
         };
     }
 
+    protected static BiFunction<Void, Throwable, Object> error(VertxTestContext testContext) {
+        return (unused, throwable) -> {
+            if (throwable != null) {
+                testContext.completeNow();
+            } else {
+                testContext.failNow(new IllegalStateException("URL reachable"));
```
Comment (jhaeyaert) [inline src/test/java/io/gravitee/notifier/webhook/AbstractWebhookNotifier.java]: Probably better to change it into "An error was expected but none has been thrown" wdyt?

**Review by jhaeyaert — APPROVED** [review]
Just a minor suggestion

**Review by loriepisicchio — APPROVED** [review]

File: src/test/java/io/gravitee/notifier/webhook/AbstractWebhookNotifier.java (hunk 1 above)
Comment (ytvnr) [reply src/test/java/io/gravitee/notifier/webhook/AbstractWebhookNotifier.java]: sure! I fp

**Review by gaetanmaisse — APPROVED** [review]

**Pull request merged**